tools/sts-job-manager/README.md

```diff
@@ -0,0 +1,195 @@
+# STS Job Manager
+
+Transferring large amounts of data, in the order of petabytes, between buckets can prove a challenge; from listing the objects, tracking the progress of the data transfer, to validating the data's integrity in the new bucket. This tool kit aims to make this task easier and relatively straightforward.
+
+## What's Included
+
+[`sts_job_manager.py`](./sts_job_manager.py)
+
+- The main tool. This tool creates STS Jobs and records each job's state.
+
+[`pause_all_jobs.py`](./pause_all_jobs.py)
+
+- A tool for pausing all running STS Transfer Operations. This is useful for controlling the output of. The [`sts_job_manager.py`](./sts_job_manager.py) automatically resumes any paused operations during it's job management cycle.
+
+[`prepare_tables.py`](./prepare_tables.py)
+
+- This creates the dataset and tables if they do not exist. It can also load the job table with a list of prefixes.
+
+[`remove_successful_one_time_jobs.py`](./remove_successful_one_time_jobs.py)
+
+- A tool for removing successful one-time jobs with complete, successful, non-running operations.
+
+## Getting Started
+
+### Assign the Appropriate Permissions
+
+#### STS Job Manager Permissions
+
+Ensure the [Storage Transfer API](https://console.developers.google.com/apis/api/storagetransfer.googleapis.com/overview) is enabled.
+
+Ensure the Storage Transfer Service service account has the [required permissions](https://cloud.google.com/storage-transfer/docs/configure-access).
+
+This tool requires an account with the following permissions:
+
+- `bigquery.datasets.create`
+- `bigquery.tables.create`
+- `bigquery.tables.getData`
+- `bigquery.tables.updateData`
+- `storagetransfer.jobs.create`
+- `storagetransfer.jobs.list`
+- `storagetransfer.operations.list`
+- `storagetransfer.operations.pause`
+- `storagetransfer.operations.resume`
+
+The additional permissions are required if monitoring is enabled:
+
+- `monitoring.metricDescriptors.create`
+- `monitoring.metricDescriptors.get`
+- `monitoring.metricDescriptors.list`
+- `monitoring.monitoredResourceDescriptors.get`
+- `monitoring.monitoredResourceDescriptors.list`
+- `monitoring.timeSeries.create`
+
+#### Pause All Jobs Permissions
+
+Requires an account with the following permissions:
```

Comment:
Please be precise in this readme file with what do you mean by account ?

Reply:
Updated and clarified in the above block